Firmware/ultralcd.cpp

```diff
@@ -5261,7 +5263,11 @@ static void lcd_main_menu()
             }
         }
     }
-    if((printJobOngoing() || printingIsPaused()) && (custom_message_type != CustomMsg::MeshBedLeveling) && !processing_tcode) {
+    if((printJobOngoing()
+        || printingIsPaused()
+        || (eeprom_read_byte((uint8_t*)EEPROM_UVLO) != PowerPanic::NO_PENDING_RECOVERY))
```

Comment:
Not sure about this `eeprom_read_byte((uint8_t*)EEPROM_UVLO) != PowerPanic::NO_PENDING_RECOVERY`.

Doesn't this mean you can _stop_ a print while power panic recovery is in progress? Maybe I'm missing something here.

In my opinion a print job can only be stopped if it is _ongoing_ or _paused_.

Reply:
You can't get out of the power panic recovery loop in case the print has failed and you DON'T want to continue.

Reply:
Note this will also affect SD prints (no Host involved). When you click "Stop print" while the printer is still recovering the print, does the printer stop correctly? If so then we can resolve this thread.

Does `EEPROM_UVLO` get reset to `NO_PENDING_RECOVERY`?

Reply:
Looked at the code in more detail and I can see that `cancel_saved_printing()` resets `EEPROM_UVLO` when the print is stopped. 👍